parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -435,40 +435,29 @@ try
         Get-HnsPsm1 -HNSModule $global:HNSModule
         Import-Module $global:HNSModule
 
-        if ($global:NetworkPlugin -eq "azure") {
-            Write-Log "Installing Azure VNet plugins"
-            Install-VnetPlugins -AzureCNIConfDir $global:AzureCNIConfDir `
-                -AzureCNIBinDir $global:AzureCNIBinDir `
-                -VNetCNIPluginsURL $global:VNetCNIPluginsURL
-
-            Set-AzureCNIConfig -AzureCNIConfDir $global:AzureCNIConfDir `
-                -KubeDnsSearchPath $global:KubeDnsSearchPath `
-                -KubeClusterCIDR $global:KubeClusterCIDR `
-                -KubeServiceCIDR $global:KubeServiceCIDR `
-                -VNetCIDR $global:VNetCIDR `
-                -IsDualStackEnabled $global:IsDualStackEnabled
-
-            if ($TargetEnvironment -ieq "AzureStackCloud") {
-                GenerateAzureStackCNIConfig `
-                    -TenantId $global:TenantId `
-                    -SubscriptionId $global:SubscriptionId `
-                    -ResourceGroup $global:ResourceGroup `
-                    -AADClientId $AADClientId `
-                    -KubeDir $global:KubeDir `
-                    -AADClientSecret $([System.Text.Encoding]::ASCII.GetString([System.Convert]::FromBase64String($AADClientSecret))) `
-                    -NetworkAPIVersion $NetworkAPIVersion `
-                    -AzureEnvironmentFilePath $([io.path]::Combine($global:KubeDir, "azurestackcloud.json")) `
-                    -IdentitySystem "{{ GetIdentitySystem }}"
-            }
-        }
-        elseif ($global:NetworkPlugin -eq "kubenet") {

```

Comment:
The intention is to remove support of kubenet and else?
But IIRC, kubenet will be considered in the future, cc @keikhara

Reply:
Remove all unused code for now. We do not know whether these code works with kubenet in aks. They are never validated in AKS. We should add kubenet support in cse when we plan to support it.

Reply:
Please let me after you do a full validation under aks after removing the code, thanks.

Reply:
I will wait until you approve it.